hooks/pre/20-certs_update.rb

```diff
@@ -0,0 +1,66 @@
+require 'fileutils'
+require 'English'
+
+if module_enabled?('katello')
+  SSL_BUILD_DIR = param('certs', 'ssl_build_dir').value
+  CHECK_SCRIPT = `which katello-certs-check`.strip
+  
+  def mark_for_update(cert_name, hostname = nil)
+    path = File.join(*[SSL_BUILD_DIR, hostname, cert_name].compact)
+    puts "Marking certificate #{path} for update"
+    if app_value(:noop)
+      puts "skipping in noop mode"
+    else
+      FileUtils.touch("#{path}.update")
+    end
```

Comment:
I never like correlating multiple lines. Something with single lines like this has my preference:
```suggestion
    if app_value(:noop)
      puts "Marking certificate #{path} for update (noop)"
    else
      puts "Marking certificate #{path} for update"
      FileUtils.touch("#{path}.update")
    end
```

Note I kept the line mostly in tact so you are close to the original output. Alternatives are `Would mark certificate #{path} for update`.